azbus/receiver.go

```diff
@@ -345,21 +332,31 @@ func (r *Receiver) Open() error {
 	}
 
 	r.receiver = receiver
+	for j := range len(r.handlers) {
+		err = r.handlers[j].Open()
+		if err != nil {
+			return fmt.Errorf("failed to open handler: %w", err)
+		}
+	}
 	return nil
 }
 
-func (r *Receiver) Close(ctx context.Context) {
+func (r *Receiver) close_() {
```

Comment:
what is the_ dogsledding on the end of this method for ?

Reply:
ah - close is an inbuilt method in go so usually adding an underline stops the confusion